Ball of Duty Server/Persistence/Player.cs

```diff
@@ -12,11 +12,18 @@ namespace Ball_of_Duty_Server.Persistence
     using System;
     using System.Collections.Generic;
     
-    public partial class Player
+    public partial class Player 
     {
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA2214:DoNotCallOverridableMethodsInConstructors")]
+        public Player()
```

Comment:
this is not ok. it will be overriden whenever the DataModel is updated.
any non autogenerated code should take place in Player.Domain, if you believe that it does not belong in a "Domain" file. then you must further partialize the class